loops-and-arrays.js

```diff
@@ -22,6 +25,14 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // ==========================================
 
 const numbers = [2, 4, 5, 29, 38];
+for (let i = 0; i<=numbers.length-1; i++) {
+       if (numbers[i]%  2 ===0){
+        console.log(numbers[i]*2);
```

Comment:
We hebben dit ook allebei fout gedaan. We moeten de array veranderen. De console.log moet uit de scope worden gehaald  en we moeten de number [i] echt updaten: numbers[i] = numbers[i] * 2;